cedar-policy-core/src/entities.rs

```diff
@@ -90,6 +90,12 @@ impl Entities {
         }
     }
 
+    /// Is this a partial store (created with `.partial()`)
+    #[cfg(feature = "partial-eval")]
+    pub fn is_partial(&self) -> bool {
+        self.mode == Mode::Partial
+    }
```

Comment:
I don't understand this. Is it needed?

Reply:
Ok, found where it's used but I'm still confused as to why the `#[cfg(feature = "partial-eval")]` is not in the body.

Reply:
Easy enough to make this method available without the feature, and just return `false` when the feature is disabled.

Should we also make `.partial()` available when the feature is disabled?

Reply:
That's why I'm confused: Why does this code compile when `partial-eval` is not enabled? I'd assume `if cfg!(feature = "partial-eval") && v.is_partial()` to be an illegal expression since `is_partial` would be undefined in that case.

Reply:
I don't think CI tests the `protobufs` feature without the `partial-eval` feature.  I think we only test with default features, no features, and all features.

Reply:
Ok, then I'm happy with the change to make it always available.